app/models/i14y_drawer.rb

```diff
@@ -12,7 +12,13 @@ def label
   end
 
   def stats
-    I14yCollections.get(self.handle).collection rescue nil
+    I14yCollections.get(handle).collection
+  rescue StandardError => error
```

Comment:
@peggles2 , how would you feel about switching our setting for the `Style/RescueStandardError` cop from `explicit` to `implicit`:
```
Style/RescueStandardError:
  Description: 'Avoid rescuing without specifying an error class.'
  Enabled: true
  VersionAdded: 0.52
  EnforcedStyle: explicit
  # implicit: Do not include the error class, `rescue`
  # explicit: Require an error class `rescue StandardError`
  SupportedStyles:
    - implicit
    - explicit
```

I prefer `rescue => error` over `rescue StandardError => error` because it's cleaner. (Obviously it's better to rescue a more specific error, but this is a case where we might encounter a fairly wide variety of errors.)

Reply:
👍